temoa-uiuc/data_files/run_mga_scenario03.txt

```diff
@@ -8,15 +8,15 @@
 # Input File (Mandatory)
 # Input can be a .sqlite or .dat file
 # Both relative path and absolute path are accepted
---input=data_files/03_uiuc.sqlite
+--input=data_files/03_uiuc_mga.sqlite
 
 # Output File (Mandatory)
 # The output file must be a existing .sqlite file
---output=data_files/03_uiuc.sqlite
+--output=data_files/03_uiuc_mga.sqlite
 
 # Scenario Name (Mandatory)
 # This scenario name is used to store results within the output .sqlite file
---scenario=mga_scenario03
+--scenario=scenario03
```

Comment:
Same here. Is scenario03 created by the Snakefile?

Reply:
It is created by the Snakefile